src/supporting-data/citations/components/results/CitationCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card } from 'franklin-sites';
+import { FC, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { getEntryPath } from '../../../../app/config/urls';
+import { getIdKeyFor } from '../../../../shared/components/results/ResultsView';
+import { Namespace } from '../../../../shared/types/namespaces';
+import { CitationsAPIModel } from '../../adapters/citationsConverter';
+import LiteratureCitation from '../LiteratureCitation';
+
+const CitationCard: FC<{
+  data: CitationsAPIModel;
+  selected?: boolean;
+  handleEntrySelection?: (rowId: string) => void;
+}> = ({ data, selected, handleEntrySelection }) => {
+  const history = useHistory();
+
+  const key = getIdKeyFor(Namespace.citations)(data);
+
+  const handleCardClick = useCallback(() => {
```

Comment:
See implementation inb `UniProtKBCard` to avoid handling clicks on links, buttons, or inputs

Reply:
See the onClick on the checkbox below, which is not needed anymore

Reply:
Actually wondering...should this logic be in the Card component itself?

Reply:
I thing we need eventually need a wrapper ResultCard component within the uniprot-website codebase that would abstract the checkbox, the selectedEntry logic, and this.
Maybe a Franklin Card can stick to being a UI component.

Reply:
Being clickable is part of the definition of the `Card` component, and we don't know whether the consumer of `Card` will have other clickable elements within it, so I'm erring more towards having this logic in Card (maybe with a prop to overwrite the set of `BLOCK_CLICK_ON_CARD`). But not for now anyway

Reply:
There's more to it though, I commented to Dan already that if we have list with checkboxes, we also need to eventually handle them in context as a user would expect them to behave. That is, click checkbox, then ctrl-click another checkbox, then all the checkboxes in between need to be clicked. This also needs to be handled somewhere.